os_stub/cryptlib_null/pk/x509.c

```diff
@@ -748,6 +748,8 @@ int32_t libspdm_x509_compare_date_time(const void *date_time1, const void *date_
  * @param[in]      requester_info        requester info to gen CSR
  * @param[in]      requester_info_length The len of requester info
  *
+ * @param[in]       is_ca                if ture,set basic_constraints ture; Otherwise, set to false.
```

Comment:
here change to ' if **true**, set basic_constraints: **CA**: ture ', more accurate.